### DISCUSSION TITLE
Report Windows : customizing width ?

### DISCUSSION THREAD

**Discussion by Derf000** [almarklein/timetagger#474] [Q&A] [answered]
Hello,
Is it possible to increase the width of the report window (or make report windows resizable) ?

With descriptions that are more or less long, they sometimes become difficult to 'read'.

If not, could "make report windows resizable" be added to the list of planned changes?

Thanks in advance

![ttagger_report](https://github.com/almarklein/timetagger/assets/112063540/17e52be0-bff6-4501-89b3-67e78dbd2d74)

**Comment by almarklein** [comment]
This is currently not possible, but might be a nice addition.

**Comment by almarklein** [comment, marked as answer]
See  #475
